exec/jdbc/src/test/java/org/apache/drill/jdbc/test/Drill2489CallsAfterCloseThrowExceptionsTest.java

```diff
@@ -537,7 +547,14 @@ public void testclosedPreparedStmtOfOpenConnMethodsThrowRight() {
         new ClosedPreparedStatementChecker(PreparedStatement.class,
                                            closedPreparedStmtOfOpenConn);
 
-    checker.testAllMethods();
+    //List of methods now supported
+    Set<String> methodsToSkip = new LinkedHashSet<String>() {{
```

Comment:
I meant to revert all these changes, they are not needed for now.

Reply:
Ok. These tests were failing because it we had implemented the `setMaxRows()` and `getMaxRows()` methods. These tests, it seems, expected them to throw exceptions.
With the `super.setLargeMaxRows()`, this should not be necessary anymore.